server/controllers/member/getMembers.js

```diff
@@ -1,3 +1,13 @@
-module.exports = (req, res) => {
-  res.send(' get member info');
+const { getMembers } = require('../../database/queries/members/getMembers');
+
+module.exports = (req, res, next) => {
+  const { offset } = req.params;
```

Comment:
@Fatmasiam It might be a good idea to let clients also pass `limit` in the params and use that for the LIMIT in the sql query.  This way, the client has full control over how many results are returned, and also the client knows how to get "page 2" of results.  Otherwise, the limit of 100 is a little bit of a mystery to the client, and the client doesn't know what offset it should pass to get the next page of results - the frontend dev will wonder, is it 100? is it 20? 50?